### PR TITLE
Use messages key instead of string  #trivial

### PR DESCRIPTION
**Pull request by austinwright**
What does this implement/fix? Explain your changes.
---------------------------------------------------
See #1364.

### CODE REVIEW

**Review by martinpucik — APPROVED** [review]
LGTM! 🚀

File: Example/Sources/Settings+UserDefaults.swift [hunk 1]
```diff
@@ -31,12 +31,12 @@ extension UserDefaults {
     // MARK: - Mock Messages
     
     func setMockMessages(count: Int) {
-        set(count, forKey: "mockMessages")
+        set(count, forKey: messagesKey)
```
Comment (martinpucik) [inline Example/Sources/Settings+UserDefaults.swift]:
```suggestion
        set(count, forKey: UserDefaults.messagesKey)
```

should fix the failing build

File: Example/Sources/Settings+UserDefaults.swift [hunk 2]
```diff
@@ -31,12 +31,12 @@ extension UserDefaults {
...
         synchronize()
     }
     
     func mockMessagesCount() -> Int {
-        if let value = object(forKey: "mockMessages") as? Int {
+        if let value = object(forKey: messagesKey) as? Int {
```
Comment (martinpucik) [inline Example/Sources/Settings+UserDefaults.swift]:
```suggestion
        if let value = object(forKey: UserDefaults.messagesKey) as? Int {
```

^^

Comment (austinwright) [reply Example/Sources/Settings+UserDefaults.swift]: Yep - I was already making the change when you commented 😛

**Pull request merged**